backend-k8s/steward-system/101-customResourceDefinition_PipelineRun.yaml

```diff
@@ -85,30 +85,24 @@ spec:
               pattern: '^run|abort$'
             logging: ###
               type: object
-              required:
```

Comment:
Add `// +optional` to pkg/apis/steward/v1alpha1/run_types.go

Reply:
Done in #43